config/fulcio-config.yaml

```diff
@@ -52,6 +52,10 @@ data:
             "https://oidc.prod-aks.azure.com/*": {
```

Comment:
Do you have any documentation from Azure about this? Is there a migration ongoing? I would assume this has to do with providing regionalized OIDC providers.

Reply:
There are a few references in the Azure org to this new URL - https://cs.github.com/?scopeName=All+repos&scope=&q=oic.prod-aks.azure.com+org%3Aazure

Contrast to https://cs.github.com/?scopeName=All+repos&scope=&q=https%3A%2F%2Foidc.prod-aks.azure.com+org%3Aazure, with more references in code. 

I can't find any public documentation about this, but seems fine to me.

Reply:
Unfortunately I couldn't find any documentation on this. I actually don't have much more information than what you could find (I'm not associated with Azure/Microsoft).

The only thing I do know is that:

- I created the AKS cluster on December 20th in the East US region, following the same steps referenced in https://github.com/sigstore/fulcio/pull/384 ([these docs](https://learn.microsoft.com/en-us/azure/aks/cluster-configuration#oidc-issuer-preview))
- The OIDC issuer URL that I got is different from what's configured here: `https://eastus.oic.prod-aks.azure.com/****`

I posted a question on their discussion forum (https://learn.microsoft.com/en-us/answers/questions/1162598/what-is-the-canonical-format-of-the-oidc-issuer-ur). Hopefully someone from Microsoft can answer and give us some concrete info we can rely on!

Reply:
Also posted on their main AKS issue for the feature (since that pings the right team): https://github.com/Azure/AKS/issues/1480#issuecomment-1397808462

Reply:
UPDATE: I got a response from Microsoft support on [the learn Q&A ticket](https://learn.microsoft.com/en-us/answers/questions/1162598/what-is-the-canonical-format-of-the-oidc-issuer-ur) which isn't definitive, but is another data point that the format appears to be `https://REGION.oic.prod-aks.azure.com/***`.

Is that sufficient data for adding this in? If not, is there something else I can do to unblock myself (e.g., is there a way for me to run a custom fulcio instance and link with the public rekor. I think in theory that's possible but not sure...)?

FWIW, I've also opened [a ticket on the Microsoft docs page](https://github.com/MicrosoftDocs/azure-docs/issues/104085) requesting for the format to be documented.